CHANGELOG.md

```diff
@@ -10,21 +10,20 @@
 
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## 0.7.0 - 2018-02-20
 * ### ALL
-    * #### Added
     * #### Changed
-        * Option string can now be a python dictionary instead of a string. It will be converted as needed (Fix #661)
+        * Option string can now be a python dictionary instead of a string. It will be converted as needed (Fix [#661](https://github.com/ni/nimi-python/issues/661))
```

Comment:
"It will be converted as needed" is in my opinion an implementation detail, can be removed from here.